Makefile

```diff
@@ -122,11 +124,21 @@ build-itest-race:
 	@$(call print, "Building itest binary for ${backend} backend.")
 	CGO_ENABLED=0 $(GOTEST) -v ./itest -tags="$(DEV_TAGS) $(RPC_TAGS) integration $(backend)" -c -o itest/itest.test$(EXEC_SUFFIX)
 
-#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin
+#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin, generate and install man pages.
 install:
 	@$(call print, "Installing lnd and lncli.")
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lnd
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lncli
+	# Invoke the 'manpages' target after installing binaries.
+	make manpages
```

Comment:
ok so the one thing here is that since our cli commands contain lots of examples that include "< ... >", the `go-md2man` tool sees these as HTML span tags and so this command then spits out a bunch of these warnings:

`WARNING: go-md2man does not handle node type HTMLSpan`

(i've confirmed that this is the issue cause I've tested removing all comments with `< .. >` & this does resolve the issue)

So I think we should figure out a way to get rid of this warning. Either we need to convert all the `<..>` to `[...]` but this might be confusing to a user - they might think that the square brackets are required.... Other option is to find some way to swallow this warning.

Reply:
I’ve done some research on this here are my two cents:

- Currently, the code related to detecting HTML blocks in the `go-md2man` tool is located in [`roff.go`](https://github.com/cpuguy83/go-md2man/blob/c2c0656178869720a1d9b013ed881db8b7a191ae/md2man/roff.go#L181-L185) file and is used by `github.com/urfave/cli` package when running `ToMan()` and swallowing these warnings requires a change to `go-md2man` which may not feasible in the meantime.
- I've found this warning on a few GitHub issues e.g this issue on [spf13/cobra](https://github.com/spf13/cobra/issues/1137) package and the way to be solved is to replace `<...>` with `[...]` as you said also he put this IBM reference [Reading command-line syntax](https://www.ibm.com/docs/en/iis/8.5?topic=home-reading-command-line-syntax) and also many issues solved this problem by this replacement.

Reply:
im not sure we should switch to `[]` because usually when we have `[]` we want the user to actually put the `[]` characters in the command. Whereas with `<>` it is implied that these should not be in the final command. 

I still think you can swallow the warning with some bash trickery. 
Something something:

```
your-command 2>&1 | grep -v "go-md2man does not handle node type HTMLSpan" || true
```

Reply:
Aha thanks a lot, @ellemouton I thought swallowing it on the package level not bash level 🙏

Reply:
Thanks. Addressed!